src/AnimateQuotes.tsx

```diff
@@ -1,18 +1,30 @@
-import React, { useRef, useEffect, useState } from 'react';
-import { Animated, Text, View } from 'react-native';
-import { QUOTES_IN_DURATION, QUOTES_IN_DELAY, QUOTES_OUT_DURATION, quotes, TOP_POSITION_PERCENT } from './constants';
+import React, { useRef, useState } from 'react';
+import { Animated } from 'react-native';
+import { QUOTES_IN_DURATION, QUOTES_IN_DELAY, QUOTES_OUT_DURATION, quotes } from './constants';
 import { getRandomInt } from './utils';
 import Quotes from "./Quotes";
 
-export default function AnimateQuotes({ isShakeTriggered, setIsShakeReady, hasInitialTransitionRun, setHasInitialTransitionRun, height }) {
+type AppPropsAnimateQuotes = {
+  isShakeTriggered: boolean,
+  setIsShakeReady: React.Dispatch<React.SetStateAction<boolean>>,
+  hasInitialTransitionRun: boolean,
+  setHasInitialTransitionRun: React.Dispatch<React.SetStateAction<boolean>>,
+  height: number
+}
+
+export default function AnimateQuotes({
+  isShakeTriggered,
+  setIsShakeReady,
+  hasInitialTransitionRun,
+  setHasInitialTransitionRun,
+  height
+}: AppPropsAnimateQuotes) {
   const [isQuoteAnimationRunning, setIsQuoteAnimationRunning] = useState(false);
   const [quoteIndex, setQuoteIndex] = useState(0);
 
   const fadeAnim = useRef(new Animated.Value(0)).current;
   const sizeAnim = useRef(new Animated.Value(0.5)).current;
 
-  const topPosition = height * TOP_POSITION_PERCENT;
```

Comment:
Committing to centered `AnimateQuotes` component no longer requires this positioning variable.